backend/src/akvo/lumen/component/flow.clj

```diff
@@ -0,0 +1,56 @@
+(ns akvo.lumen.component.flow
+  "wrap flow and flow api logic and data related"
+  (:require [integrant.core :as ig]
+            [akvo.lumen.component.keycloak :as c.keycloak]
+            [clj-http.client :as http]
+            [clojure.tools.logging :as log]
+            [clojure.spec.alpha :as s]))
+
+(defn access-token
```

Comment:
from line 9 to line 28, this code comes from backend/src/akvo/lumen/lib/import/flow_common.clj
so just wrapping flow stuff into flow component